README.md

```diff
@@ -0,0 +1,93 @@
+# Fredhopper/SAP Hybris Connector
+
+*Powerful on-site search, navigation, personalisation and merchandising, tailored to the specific needs of leading retailers*
+
+![](Fredhopper_logo.jpg)
+
+With the Fredhopper *Connector for SAP Hybris*, you can leverage the advanced on-site search, navigation, personalisation, and merchandising capabilities of Fredhopper in your hybris-based online marketplace.
+
+* [System Requirements](#system-requirements)
+* [Installation and Configuration](#installation-and configuration)
+	* [Install the Extension](#install-the-extension)
+	* [Configure the Export Logic](#configure-the-export-logic)
+	* [Configure the Query Logic](#configure-the-query-logic)
+* [License](#license)
+
+## System Requirements
+
+* SAP Hybris 6.x
+* Configured Fredhopper instance<br>The machine on which you are running hybris must be able to reach the Fredhopper instance.
+
+## Installation and Configuration
+
+To be able to use the Fredhopper/SAP Hyris connector, you need to build and install the extension locally and configure the export and query logic.
+
+### Install the Extension
+
+Before you can use the Fredhopper/SAP Hybris connector, you need to build the custom extension locally and add it to hybris.
+
+1. Clone the [Fredhopper GitHub repository](https://github.com/fredhopper/hybris-connector.git) to your local machine.
```

Comment:
I'd go with the "Fredhopper/SAP Hybris Connector repository". This way we'll incorporate the product name that marketing are using right now. Now capitalizing repository is a deliberate choice, since capitalizing it would imply that this is a separate product or a trademark.